### PR TITLE
[filesystems] Do not repeat section title in cross-references to [fs.…

### LINKED ISSUE #1116: Replace "Error reporting (27.10.7)" with just "27.10.7" throughout [filesystems]
[filesystems] contains cross-references like this:

> *Throws:* As specified in Error reporting (27.10.7).

This should instead be simply

> *Throws:* As specified in 27.10.7.

We do not repeat section names in cross-references to those sections.

### PR DESCRIPTION
**Pull request by jensmaurer**
…err.report].

Fixes #1116.

### CODE REVIEW

**Comment by zygoloid** [comment]
We have one more of these to fix now. Can you rebase fix the new one too?

**Comment by jensmaurer** [comment]
Rebased.

File: source/iostreams.tex [hunk 1]
```diff
@@ -13262,7 +13262,7 @@
 \begin{note} Because hardware failures, network failures, file system races~(\ref{fs.def.race}),
 and many other kinds of errors occur frequently in file system operations, users should be aware
 that any filesystem operation function, no matter how apparently innocuous, may encounter
-an error. See Error reporting~(\ref{fs.err.report}). \end{note}
+an error; see~(\ref{fs.err.report}). \end{note}
```
Comment (tkoeppe) [inline source/iostreams.tex]: No parentheses?

Comment (jensmaurer) [inline source/iostreams.tex]: Agreed.

**Pull request merged**